resources/styles/components/task-step/all-steps.less

```diff
@@ -36,6 +36,11 @@
       }
     }
 
+    .interactive-content > .note {
+      iframe.interactive {
+        margin-left: -160px;
```

Comment:
Could you calculate this offset based on variables for the note width or put in a comment to clarify why this number was chosen so someone else looking at it a month from know would know why?